features/support/run_helpers.rb

```diff
@@ -91,5 +91,8 @@ def should_print_command_output? command, debug
 
 # Output of last `run` without text formatting (ANSI escape sequences)
 def unformatted_last_run_output
-  @last_run_result.out.gsub(/\e[^m]+m/, '')
+  @last_run_result.out
+    .gsub(/\e[^m]*m/, '')   # remove Bash colors
```

Comment:
`# remove ANSI escape sequences` instead of  `# remove Bash colors`
Also please remove `(ANSI escape sequences)` from the method comment